install.bash

```diff
@@ -32,7 +32,6 @@ SCRATCH_MONGODB_INSTALL=false
 BUILD_APP=true
 VOXEL51_INSTALL=false
 # Only install Python dependencies, skip git setup
-DEPS_ONLY=false
```

Comment:
Reverts part of the change in #18. The `DEPS_ONLY` flag was added because `/src` wasn't available during build (and thus `pip install -e .` failed), but now it is.